perun-wui-userprofile/src/main/java/cz/metacentrum/perun/wui/userprofile/client/UserProfileView.ui.xml

```diff
@@ -6,16 +6,13 @@
 
 	<ui:with field="res" type="cz.metacentrum.perun.wui.userprofile.client.resources.UserProfileResources"/>
 
-	<ui:style>
-		.bottom { border-radius: 0px !important; }
-		.minWidthTop { min-width: 1200px; margin-bottom: 0px !important; }
```

Comment:
I don't know the reason why it was here. But with it a collapse button wasn't visible on small devices. Without it everything seems fine.